### PR TITLE
Add user currency to export

### PR DESCRIPTION
**Pull request by christoph93**

### CODE REVIEW

File: apps/api/src/app/export/export.service.ts [hunk 1]
```diff
@@ -95,7 +99,10 @@ export class ExportService {
                 : SymbolProfile.symbol
           };
         }
-      )
+      ),
+      user: {
+        settings: { currency: userWithSettings.Settings.settings.baseCurrency }
```
Comment (dtslvr) [inline apps/api/src/app/export/export.service.ts:104]: You can use `userCurrency`, then you don't even have to make a query 🙂

```
settings: { currency: userCurrency }
```

**Review by dtslvr — CHANGES_REQUESTED** [review]
Thank you for the contribution @christoph93! 😎

I have found an easier way to access the currency of the user. Can you please revert everything that is not needed?

File: apps/api/src/app/export/export.service.ts (hunk 1 above)
Comment (christoph93) [reply apps/api/src/app/export/export.service.ts]: Fixed!

**Review by dtslvr — APPROVED** [review]
Thank you! 👍🏻
Ready to merge now.

**Pull request merged**